deployment/data-streams/changeset/verifier-proxy/v0.5.0/call_set_access_controller_test.go

```diff
@@ -0,0 +1,68 @@
+package v0_5_0
+
+import (
+	"testing"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	commonChangesets "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/testutil"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_proxy_v0_5_0"
+)
+
+func TestSetAccessController(t *testing.T) {
+	e := testutil.NewMemoryEnv(t, true)
+	acAddress := common.HexToAddress("0x0000000000000000000000000000000000000123")
+	testChain := e.AllChainSelectors()[0]
+	resp, err := commonChangesets.Apply(t, e, nil,
+		commonChangesets.Configure(
+			DeployVerifierProxyChangeset,
+			DeployVerifierProxyConfig{
+				ChainsToDeploy: []uint64{testChain},
+				Version:        *semver.MustParse("0.5.0"),
+			},
+		),
+	)
+	require.NoError(t, err)
+
+	// Ensure the VerifierProxy was deployed
+	ab, err := resp.ExistingAddresses.Addresses()
+	require.NoError(t, err)
+	require.Len(t, ab, 1)
+
+	verifierProxyAddrHex, err := deployment.SearchAddressBook(resp.ExistingAddresses, testChain, types.VerifierProxy)
+	require.NoError(t, err)
+	verifierProxyAddr := common.HexToAddress(verifierProxyAddrHex)
+
+	cfg := VerifierProxySetAccessControllerConfig{
+		ConfigPerChain: map[uint64]SetAccessControllerConfig{
+			testChain: {
+				AccessControllerAddress: acAddress,
+				ContractAddress:         verifierProxyAddr,
+			},
+		},
+	}
+
+	err = e.ExistingAddresses.Merge(resp.ExistingAddresses)
```

Comment:
Usually I am doing 
`e, err := commonChangesets.Apply(t, e...`
As Apply returns the updated environment

Reply:
thanks updated - test was outdated